erpnext_shopify/patches/V1_0/set_variant_id.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+from erpnext_shopify.utils import get_shopify_items
+from frappe.utils import cint
+from erpnext_shopify.exceptions import ShopifyError
+import requests.exceptions
+
+def execute():
+	shopify_settings = frappe.get_doc("Shopify Settings")
+	shopify_items = get_item_list()
+	
+	if shopify_settings.shopify_url and shopify_items:
+		for item in frappe.db.sql("""select name, item_code, shopify_id, has_variants, variant_of from tabItem 
+			where sync_with_shopify=1 and shopify_id is not null""", as_dict=1):
+			
+			if item.get("variant_of"):
+				frappe.db.sql(""" update tabItem set variant_id=shopify_id 
```

Comment:
This fieldname should be `shopify_variant_id`. Also you need to reload Item doctype before running this patch.